src/authorizations/actions/thunks.ts

```diff
@@ -168,6 +168,30 @@ export const deleteAuthorization = (id: string, name: string = '') => async (
   }
 }
 
+export const bulkDeleteAuthorizations = (tokenIds: string[]) => async (
+  dispatch: Dispatch<Action | NotificationAction>
+) => {
+  try {
+    for (const id of tokenIds) {
+      try {
+        const resp = await api.deleteAuthorization({authID: id})
+
+        if (resp.status !== 204) {
+          throw new Error(resp.data.message)
+        }
+        event('token.delete.success', {id})
+        dispatch(removeAuthorization(id))
+      } catch (e) {
+        event('token.delete.failure', {id})
```

Comment:
what's the value added here for including the ID in the event? This type of pattern creates a unique value for every event, id event. I don't see the point, is this a product ask or is simply knowing that a token has been deleted valuable enough?

Reply:
This is true, that was a copy paste error / leftover. I have it fixed and will push soon!